### PR TITLE
Add support for azure pipeline tests

### PR DESCRIPTION
**Pull request by narph**
## What does this PR do?

Add support for azure pipeline tests

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [ ] I have verified that all datasets collect metrics or logs.

## Related issues

- Closes #123

### CODE REVIEW

File: packages/azure/data_stream/auditlogs/fields/ecs.yml [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  description: Unique identifier of the user.
+- description: Short name or login of the user.
+  example: albert
+  ignore_above: 1024
+  multi_fields:
+    - flat_name: user.name.text
+      name: text
+      norms: false
+      type: text
+  name: user.name
```
Comment (mtojek) [inline packages/azure/data_stream/auditlogs/fields/ecs.yml:240]: nit: hm... `user.name` is also defined in line 226. I think you can leave one definition.

File: packages/azure/data_stream/platformlogs/_dev/test/pipeline/test-platformlogs-invalid-raw.log-expected.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+            "event": {
+                "ingested": "2020-11-26T13:17:42.047220600Z"
+            },
+            "error": {
+                "message": "Unexpected character ('\\\\' (code 92)): was expecting double-quote to start field name\\n at [Source: (org.elasticsearch.common.io.stream.InputStreamStreamInput); line: 1, column: 4]"
```
Comment (mtojek) [inline packages/azure/data_stream/platformlogs/_dev/test/pipeline/test-platformlogs-invalid-raw.log-expected.json:9]: Negative tests.. nice!

File: packages/azure/data_stream/platformlogs/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -135,6 +135,10 @@ processors:
       field: azure.platformlogs.properties.statusCode
       target_field: azure.platformlogs.properties.status_code
       ignore_missing: true
+  - rename:
+      field: azure.platformlogs.Status
```
Comment (mtojek) [inline packages/azure/data_stream/platformlogs/elasticsearch/ingest_pipeline/default.yml:139]: nit: `azure.platformlogs.Status` or `azure.platformlogs.status`?

**Review by mtojek — APPROVED** [review]
Ship it!

File: packages/azure/data_stream/platformlogs/elasticsearch/ingest_pipeline/default.yml (hunk 3 above)
Comment (narph) [reply packages/azure/data_stream/platformlogs/elasticsearch/ingest_pipeline/default.yml:139]: `azure.platformlogs.status`

**Pull request merged**